class2.md

```diff
@@ -0,0 +1,287 @@
+
+**Reality/truth:** a persons health history, location, occupation, diet, habits, stress levels, among many other things will play into if someone gets cardiovascular disease. 
+
+**Dataset:** when creating the dataset that will be used to make our predictions we will never capture the whole truth. We will capture as much of the truth as we can by collecting data on various features that we believe are related to the problem we are trying to solve. Many underlying factors that lead to the cardiovascular disease may be unknown or we might not be able to measure them or capture them in our dataset.
+
+**Features/variables:** These are the measurable data that make up our dataset. Some features collected might be useless while others might carry a substantial amount of weight in making the prediction. Since this is a training set one of the variables included is the label that we are trying to predict (`cvd`).
+
+**Inputs:** Features that we have collected and will use to make our prediction. In this dataset our inputs would be all of the columns except `patient_Id` and `cvd`. We would likely remove `pateint_Id` because a random identifier won't have any predictive power and `cvd` would not be included as an input since it is the target we are trying to predict.
+
+**Output:** The variable that we are trying to predict. In this case `cvd`.
+
+### An overview of supervised learning
+
+<p align="center">
+  <img align="middle" width="400" src="images/supervised.png">
+</p>
+
+The goal of supervised machine learning is to fit a model that relates response variables to predictors and can accurately predict the response for future observations of predictor variables. We will cover supervised learning more in depth in class 2 of this series.
+
+Supervised learning is one of the most commonly used forms of machine learning and it's generally easier to implement and interpret than unsupervised learning alogrithms. 
+
+A hallmark of supervised learning algorithms is that they work in two steps: training and testing. This requires training the algorithm on example datasets with inputs that are annotated (or labled) with the correct response. Once training is complete the algorithm will be tested on a seperate labled dataset and it's performance is evaluated.
+
+Ideally an the algorithm will be specific enough to accurately predict the response variable, but generalizable so that it works with new datasets.
+
+There are two subclasses of supervised learning:
+1. Regression: predicting a numeric
+2. Classification: predicting a category
+
+### Some basic examples of supervised machine learning
+
+#### Determining if a tumor is benign or not from an image
+
+<p align="center">
+  <img width="350" alt="" src="images/histopathBreast.jpg">
+</p>
+
+* Input: Images of tumors
+* Output: Binary; begign or malignant
+* Dataset: Need a database of medical images, expert diagnosis of benign or malignant
+
+#### Identifying handwritten letters
```

Comment:
Switch the order of these examples, so the non-biology (e.g, more general) example is first

Reply:
begign -> benign